CHANGELOG.md

```diff
@@ -4,10 +4,46 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.72] - Unreleased
+
+### Changed
+
+- Update `netflow` plugin ([PR316](https://github.com/observIQ/stanza-plugins/pull/316))
+  - Added `listen_ip` as parameter
+  - Added `listen_port` as parameter
+  - Changed description of `listen_address` to "Parameter Deprecated Use `listen_ip` and `listen_port` instead."
+- Update `sflow` plugin ([PR316](https://github.com/observIQ/stanza-plugins/pull/316))
+  - Added `listen_ip` as parameter
+  - Added `listen_port` as parameter
+  - Changed description of `listen_address` to "Parameter Deprecated Use `listen_ip` and `listen_port` instead."
+- Update `vmware_esxi` plugin ([PR316](https://github.com/observIQ/stanza-plugins/pull/316))
+  - Added `listen_ip` as parameter
+  - Added `listen_port` as parameter
+  - Changed description of `listen_address` to "Parameter Deprecated Use `listen_ip` and `listen_port` instead."
+- Update `vmware_vcenter` plugin ([PR316](https://github.com/observIQ/stanza-plugins/pull/316))
+  - Added `listen_ip` as parameter
+  - Added `listen_port` as parameter
+  - Changed description of `listen_address` to "Parameter Deprecated Use `listen_ip` and `listen_port` instead."
+  - Added fall back regex parser incase syslog parser fails. This will attempt to parse severity from priority. The unparsed portion will be put in message.
+- Update `ubiquiti` plugin ([PR316](https://github.com/observIQ/stanza-plugins/pull/316))
+  - Added `listen_ip` as parameter
+  - Added `listen_port` as parameter
+  - Changed description of `listen_address` to "Parameter Deprecated Use `listen_ip` and `listen_port` instead."
+- Update `cisco_meraki` plugin ([PR316](https://github.com/observIQ/stanza-plugins/pull/316))
+  - Added `listen_ip` as parameter
+  - Added `listen_port` as parameter
+  - Changed description of `listen_address` to "Parameter Deprecated Use `listen_ip` and `listen_port` instead."
+- Update `syslog`, `rsyslog`, `syslogng` plugin ([PR316](https://github.com/observIQ/stanza-plugins/pull/316))
+  - Added `listen_ip` as parameter
+  - Added `listen_port` as parameter
+  - Added parameter `advanced_config` set to true on `listen_ip` and `location`. This is to be used in UI to hide `listen_ip` behind an advanced option
+  - Changed description of `listen_address` to "Parameter Deprecated Use `listen_ip` and `listen_port` instead."
+  - Added fall back regex parser incase syslog parser fails. This will attempt to parse severity from priority. The unparsed portion will be put in message.
```

Comment:
Can the size of this be reduced? Perhaps group all the identical changes?

Reply:
Reduced all syslog entries to one list.